sdk/eventhub/Azure.Messaging.EventHubs/src/Properties/AssemblyInfo.cs

```diff
@@ -2,6 +2,13 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System.Runtime.CompilerServices;
 using Azure.Core;
 
 [assembly: AzureSdkClientLibraryAttribute("eventhubs")]
+
+#if CODESIGN
```

Comment:
Do we still need this in Track2 SDKs?

Reply:
That's a fair question, now that we're including the full key in the repo and not delay-signing.   I'll double-check and likely yank this out.